README.md

```diff
@@ -1,28 +1,41 @@
 # MolEvolvR Stack
 
-This repo contains the implementation of the MolEvolvR stack, i.e.:
+This repo contains the implementation of the MolEvolvR stack, consisting of:
 - `frontend`: the frontend web app, written in React
 - `backend`: a backend written in [Plumber](https://www.rplumber.io/index.html)
 - `cluster`: the containerized SLURM "cluster" on which jobs are run
 - `services`: a collection of services on which the stack relies:
     - `postgres`: configuration for a PostgreSQL database, which stores job information
 
-Most of the data processing is accomplished via the `MolEvolvR` package, which
-is currently available at https://github.com/JRaviLab/molevolvr. The stack
-simply provides a user-friendly interface for accepting and monitoring the
-progress of jobs, and orchestrates running the jobs on SLURM. The jobs
-themselves call methods of the package at each stage of processing.
+Most of the data processing is accomplished via the `MolEvolvR` package, available at https://github.com/JRaviLab/molevolvr. The stack provides a user-friendly interface for accepting and monitoring job progress, and orchestrates running the jobs on SLURM. The jobs themselves call methods of the package at each stage of processing.
 
-## Running the Stack in Development
+## Prerequisites
 
 To run the stack, you'll need to [install Docker and Docker Compose](https://www.docker.com/).
 
-First, copy `.env.TEMPLATE` to `.env` and fill in the necessary values. You
+## Setup
+
+1. Fork and Clone this repository:
+   git clone <https://github.com/JRaviLab/molevolvr2.0.git> then run this command
+
+      `cd <your-repository-directory>`
+   to move into your cloned directory
```

Comment:
I don't see where forking comes up in your directions since you clone the original repo, and I don't think I want to recommend that people fork the repo anyway unless they intend to contribute.

Also, I think the cloning and cd'ing could be more explicit, e.g.:

```suggestion
1. Clone this repository:
  `git clone https://github.com/JRaviLab/molevolvr2.0.git`
  Then run `cd molevolvr2.0` to move into the project directory.
```

Reply:
Alright, thanks for your comments @vincerubinetti and @falquaddoom, i will apply them and push an updated version